dockerfiles/keycloak/kc_realm_user.sh

```diff
@@ -38,6 +38,7 @@ if [ "${CHE_SELF__SIGNED__CERT}" != "" ]; then
     keytool -importcert -alias HOSTDOMAIN -keystore /scripts/openshift.jks -file /scripts/openshift.cer -storepass openshift -noprompt
     keytool -importkeystore -srckeystore $JAVA_HOME/jre/lib/security/cacerts -destkeystore /scripts/openshift.jks -srcstorepass changeit -deststorepass openshift
     /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/cli/add_openshift_certificate.cli && rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
+    /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/cli/add_openshift_certificate-ha.cli 
```

Comment:
I am not sure about this.
Should we use either `add_openshift_certificate.cli ` or `add_openshift_certificate-ha.cli ` ?

Reply:
I just add my part for my use case (Openshift deployment / Oauth with Openshift / Multi User / Self Sign Certificate) , and I am not sure if I remove the stanadalone.xml modification, I don't broke something.

I prefer add my part and modify the standalone-ha.xml with the correct trust store, I don t see all use cases.